meta-bbb/recipes-kernel/linux/linux-stable-6.13/bbb/defconfig

```diff
@@ -5711,7 +5745,24 @@ CONFIG_UBIFS_FS_XATTR=y
 CONFIG_UBIFS_FS_SECURITY=y
 # CONFIG_UBIFS_FS_AUTHENTICATION is not set
 # CONFIG_CRAMFS is not set
-# CONFIG_SQUASHFS is not set
+CONFIG_SQUASHFS=y
+# CONFIG_SQUASHFS_FILE_CACHE is not set
+CONFIG_SQUASHFS_FILE_DIRECT=y
```

Comment:
Done!
@a3f:
CONFIG_SQUASHFS_FILE_DIRECT gives better performance. Also CONFIG_SQUASHFS_CHOICE_DECOMP_BY_MOUNT is not helpful IMO, because software won't usually change the decompression type at mount time.
Just select only CONFIG_SQUASHFS_DECOMP_MULTI. (PERCPU is unnecessary overhead as you are single core).